src/main/scala/io/epiphanous/flinkrunner/model/sink/JdbcSinkConfig.scala

```diff
@@ -609,6 +604,8 @@ case class JdbcSinkConfig[ADT <: FlinkEvent](
       data: Map[String, Any],
       statement: PreparedStatement,
       element: E): Unit = {
+      val mapper = new ObjectMapper()
```

Comment:
We already have a `JsonRowEncoder[T]` in flink runner. Can we use that here instead, so we don't need to duplicate the process? This isn't a lot of code now, but I'm thinking of the future, where we discover a map that contains data that requires more elaborate configuration of this mapper.